lib/workers/global/config/parse/env.spec.ts

```diff
@@ -281,6 +281,7 @@ describe('workers/global/config/parse/env', () => {
         mergeConfidenceDatasources: ['docker'],
         autodiscoverRepoSort: 'alpha',
         autodiscoverRepoOrder: 'desc',
+        dockerMaxPages: 10,
```

Comment:
The docs say the _default_ maximum of Docker pages to fetch is `20`. But this test has `dockerMaxPages` set to `10`. Is that correct?

Reply:
Yes for this test its fine.